doc/stomp_planner/stomp_planner_tutorial.rst

```diff
@@ -0,0 +1,221 @@
+
+Installing MoveIt! from Source
+------------------------------
+As you add and remove packages from your workspace you will need to clean your workspace and re-run the command to install new missing dependencies. Clean your workspace to remove references to the system wide installation of MoveIt!: ::
+
+  cd ~/ws_moveit/src
+  catkin clean
+
+Now follow the instructions on the MoveIt! homepage for `installing MoveIt! Kinetic from source <http://moveit.ros.org/install/source/>`_. Note that you can skip the **Prerequisites** section since you should already have a Catkin workspace.
+
+Re-source the setup files: ::
+
+  source ~/ws_moveit/devel/setup.bash
+
+Using STOMP with Your Robot
+---------------------------
+**Note:** if you are following this demo using the ``panda_moveit_config`` from the `ros-planning/panda_moveit_config <https://github.com/ros-planning/panda_moveit_config>`_ repository, these steps are already done for you and you can skip steps 1-3 and you only need to do step 4.
+
+#. Simply download `stomp_planning_pipeline.launch.xml <https://github.com/ros-planning/panda_moveit_config/blob/master/launch/stomp_planning_pipeline.launch.xml>`_ file into the launch directory of your MoveIt! config package. In our case, we will save this file in the ``panda_moveit_config/launch`` directory. Place the file "*stomp_planning_pipeline.launch.xml*" file in the **launch** directory of your **moveit_config** package.  The file should contain the following: ::
+   
+    <launch>
+      <!-- Stomp Plugin for MoveIt! -->
+      <arg name="planning_plugin" value="stomp_moveit/StompPlannerManager" />
+
+      <!-- The request adapters (plugins) ORDER MATTERS -->
+      <arg name="planning_adapters" value="default_planner_request_adapters/FixWorkspaceBounds
+                                          default_planner_request_adapters/FixStartStateBounds
+                                          default_planner_request_adapters/FixStartStateCollision
+                                          default_planner_request_adapters/FixStartStatePathConstraints" />
+      <arg name="start_state_max_bounds_error" value="0.1" />
+      <param name="planning_plugin" value="$(arg planning_plugin)" />
+      <param name="request_adapters" value="$(arg planning_adapters)" />
+      <param name="start_state_max_bounds_error" value="$(arg start_state_max_bounds_error)" />
+      <rosparam command="load" file="$(find panda_moveit_config)/config/stomp_planning.yaml"/>
+    </launch>
+   
+     
+   **>** Take notice of the **stomp_planning.yaml** configuration file, this file must exists in moveit_config package.
```

Comment:
adding the yaml file should be a separate step, not just a note

Reply:
removed this line as its present in the later step (step 3)